.github/workflows/twister_dfu_nrf52840dk.yml

```diff
@@ -0,0 +1,139 @@
+name: Twister DFU nrf52840dk
+
+on:
+  push:
+    branches: [ main ]
+  schedule:
+    # Run workflow at the start of every day (12 AM UTC)
+    - cron: "0 0 * * *"
+
+  workflow_dispatch:
+
+jobs:
+  # Assumptions made about the self-hosted runner:
+  #
+  #  1. Has installed the GitHub Actions self-hosted runner service
+  #  2. Has nrfjprog already installed at /opt/nrf-command-line-tools
```

Comment:
Just a note for posterity, we should look into options for making this managed automatically (docker, conda, etc.). As we grow the number of runners and tests, managing these dependencies and making sure they're all on the same version and such will get unwieldy.